### PR TITLE
Heidi CCF 3D library

### PR DESCRIPTION
**Pull request by HeidiS25**
Changed the layout- need help to center text under icons. Not ready to merge! Need to link icons and models properly, need Katy approval. First time creating a pull request on GitHub- let me know if I did anything wrong!

### CODE REVIEW

File: pages/ccf-3d-reference-library.html [hunk 1]
```diff
@@ -119,47 +97,51 @@ <h4 class="mt-5">Reference Organs</h4>
                                 <ul class="nav nav-tabs bg-light" id="myTab" role="tablist">
                                     <li class="nav-item">
                                         <a class="nav-link active" id="bone-tab" data-toggle="tab" href="#bone"
-                                            role="tab" aria-controls="home" aria-selected="true">Bone Marrow & Blood/Pelvis</a>
+                                            role="tab" aria-controls="home" aria-selected="true"><img src= "/img/icons/ico-organs-pelvis f.svg" Width=45.7px height=45.7px ><br/>Bone Marrow &<br/> Blood/Pelvis</a>
```
Comment (bherr2) [inline pages/ccf-3d-reference-library.html]: Don't use starting / for links. Use '../img/XXXX'. When deployed to production it's under /ccf/ and thus all references to /img will fail.

File: pages/ccf-3d-reference-library.html [hunk 2]
```diff
@@ -119,47 +97,51 @@ <h4 class="mt-5">Reference Organs</h4>
                                 <ul class="nav nav-tabs bg-light" id="myTab" role="tablist">
                                     <li class="nav-item">
                                         <a class="nav-link active" id="bone-tab" data-toggle="tab" href="#bone"
```
Comment (bherr2) [inline pages/ccf-3d-reference-library.html]: A quick local check found that adding `text-align: center;` to the nav-link css definition does the trick of centering things.

**Review by bherr2 — CHANGES_REQUESTED** [review]
The icon links and content links are broken. You'll need to either upload them or reference them from the appropriate urls.

**Comment by HeidiS25** [comment]
Hi Bruce- I sent a Slack convo about this- not sure icons are still pulling, but hopefully the rest is ok. Let me know. Thanks!

**Comment by HeidiS25** [comment]
Bruce- I want to preview these latest changes to Katy- is there a way she can see it without being live to the public?

**Pull request merged**